usage/src/com/cloud/usage/UsageManagerImpl.java

```diff
@@ -1054,8 +1054,6 @@ private void createVMHelperEvent(UsageEventVO event) {
 
         if (EventTypes.EVENT_VM_START.equals(event.getType())) {
             // create a new usage_VM_instance row for this VM
-            try {
```

Comment:
Fix indentations if you want to remove try-catch, also add a test?

Reply:
marvin test case is not there.
Manually tested the scenario with and without the fix for duplicate entries.

Reply:
@jayakarteek please add a test case

Reply:
I already tested the scenario with out the fix and posted the results in above comments 
Also @niteshsarda  tested the fix and gave test LGTM.
Are you looking for marvin test case

Reply:
@rhtyd can you reply , so that I can proceed

Reply:
@jayakarteek yes, a marvin test would be great that way we can test the behaviour without needing manual QA

Reply:
@jayakarteek can you also addressed any outstanding issue on the PR, and explain some unanswered questions including - (1) how do we reproduce this error in a new master based environment, (2) instead of ignoring/stopping at the exception does it make sense to fix the cases that lead to throwing/suppression of exceptions?

Reply:
@rhtyd  
1 We cannot reproduce the scenario directly, as it depends on usage events. 
2 Duplicate usage occurs due to in correct database records and we may not know which scenarios are populating the incorrect values. 
Ideally it is good to fix the cases that are generating wrong database records.
But there may be more then one cases that causes this error, fixing all dependent systems is time consuming and complicated so better to handle here